packages/cpp/src/browser/cpp-build-configurations.ts

```diff
@@ -279,10 +310,20 @@ export class CppBuildConfigurationManagerImpl implements CppBuildConfigurationMa
         return this.activeConfigurations.get(workspaceRoot);
     }
 
+    /**
+     * Get all active build configurations.
+     *
+     * @returns the map of all active configurations for each workspace root.
```

Comment:
looks like the mapping is `one root, one config`. 

`all active configurations for each workspace root` looks more like `zero to many configs per root`.

Reply:
I've updated the documentation to be a bit clearer, please let me know if it can be improved further :)